### ISSUE TITLE
add support for websocket forwarding

### ISSUE THREAD

**Issue by jstrachan** [jimmidyson/kuisp#8]
when proxying the kubernetes REST API its looking like the websockets don't proxy yet

```
WebSocket connection to 'ws://k8s/api/v1/namespaces/default/pods?watch=true&access_token' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVED
```

it'd be nice to be able to proxy those too

**Comment by jstrachan** [comment]
I've done a spike implementing this in this branch: https://github.com/jstrachan/kuisp/tree/websocket though have TLS issues right now...

**Comment by jimmidyson** [comment]
Looks like you need to use wss rather than plain ws?

**Comment by jstrachan** [comment]
it does use wss :( https://github.com/jstrachan/kuisp/blob/websocket/kuisp.go#L158 - which then under the covers does a https GET

**Comment by jimmidyson** [comment]
Fixed in #9
